### ISSUE TITLE
Feature Request: Automatically connect to voice server announced by game server

### ISSUE THREAD

**Issue by justdcsthings** [ciribob/DCS-SimpleRadioStandalone#90]
When a player connects to the game server, a script would send them a special chat message. Another script on client-side picks up this message and automatically connects to the right voice server.

Proof of concept: https://gist.github.com/anonymous/24bf6d98ce30b00da8a2fcc9581a8f49

**Comment by ciribob** [comment]
Thanks! Alas that is only called server side, not clientside... :(

My thinking was just add a tag into the mission description as I can parse instead.

Something like

SRS:someip-or domain\n

**Comment by justdcsthings** [comment]
What do you mean, only called server-side? The script is meant to be run on both server and client.

These are the logs I get. On server:
```
00010.685 INFO   : Registered callbacks.
00128.658 INFO   : Sending auto connect message to player 2.
```

On client:
```
00012.560 INFO   : Registered callbacks.
00177.018 INFO   : Got AutoConnect message: 127.0.0.1
```

But I would be happy with putting it into the mission description as well :)

**Comment by ciribob** [comment]
oh I thought the onChatMessage was server side only in the GameGui. If not as you've shown, thats brilliant! Thanks for that as this is an easier drop-in and more reliable then the mission description that can be missing. :+1:

**Comment by ciribob** [comment]
Aiming for this to be in the next release as part of allowing the server to listen on different ports. Thanks again for the Gist!

**Comment by ciribob** [comment]
Fixed in #97

Thanks again for the gist
